pipelines/api-service/base/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+resources:
```

Comment:
please add `images` in all kustomization.yaml so that when release or upgrade, we can just update images list in  kustomization.yaml instead updating all deployment files.